data/design/agi_designers.json

```diff
@@ -0,0 +1,715 @@
+  "Javier Mariscal",
+  "Jean-Beno\u00eet L\u00e9vy",
+  "Jean Carlu",
+  "Jean Colin",
+  "Jean David",
+  "Jeanette Collins",
+  "Jean Picart Le Doux",
+  "Jean Robert",
+  "Jean Widmer",
+  "Jeffrey Fisher",
+  "Jelle Van der Toorn Vrijthoff",
+  "Jennifer Sterling",
+  "Jeremy Coysten",
+  "Jeremy Leslie",
+  "Jessica Helfand",
+  "Jessica Walsh",
+  "Jianping He",
+  "Jian Zhao",
+  "Jiaying Han",
+  "James Cross",
+  "Jingren Lu",
+  "Jennifer Morla",
+  "Jock Kinneir",
+  "Joe Hyoun Youl",
+  "John Maeda",
+  "John Massey",
+  "John McConnell",
+  "John  Morgan",
+  "Johnny Kelly",
+  "John Rushworth",
+  "Jonas Hecksher",
+  "Jonas V\u00f6geli",
+  "Jonathon Jeffrey",
+  "Jon Sueda",
+  "Joonho Kwon",
+  "Joost Grootens",
+  "Jorge Alderete",
+  "Jorge Silva",
+  "J\u00f6rg Zintzmeyer",
+  "Jos\u00e9 Albergaria",
+  "Josef Flejsar",
+  "Josef Mrosczak",
+  "Josef M\u00fcller-Brockmann",
+  "Josef Svoboda",
+  "Joseph Binder",
+  "Josep Pla-Narbona",
+  "Jozef Willibrord Swarte",
+  "Juan Pablo Fajardo Gonz\u00e1lez ",
+  "Jukka Veistola",
+  "Julia Hasting",
+  "Julian Palka",
+  "Julien Vall\u00e9e",
+  "Jun He",
+  "Jurriaan Schrofer",
+  "Justus Oehler",
+  "Kaoru Kasai",
+  "Karin Fong",
+  "Kari Piippo",
+  "Karl Maier",
+  "Karl Oskar Blase",
+  "Kashiwa Sato",
+  "Katherine McCoy",
+  "Katsuhiko Shibuya",
+  "Katsumi Asaba",
+  "Kazufumi Nagai",
+  "Kazumasa Nagai",
+  "Kazunari Hattori",
+  "K. Domenic Geissb\u00fchler",
+  "Keiko Hirano",
+  "Kei Matsushita",
+  "Keisuke Nagatomo",
+  "Keith Godard",
+  "Keizo Matsui",
+  "Ken Carbone",
+  "Ken Garland",
+  "Kenji Itoh",
+  "Kenjiro Sano",
+  "Ken Miki",
+  "Kenneth W. Cato AO",
+  "Kenya Hara",
+  "Kiko Farkas",
+  "Kit Hinrichs",
+  "Kiyoshi Awazu",
+  "Klaus Ensikat",
+  "Koichi Sato",
+  "Kris Sowersby",
+  "Kurt Weidemann",
+  "Kurt Wirth",
+  "Kyle Cooper",
+  "Ky\u00f6sti Varis",
+  "Kyuhyung Cho",
+  "Kyungsun Kymn",
+  "Lam Hung",
+  "Lance Wyman",
+  "Lanny Sommese",
+  "Lars M\u00fcller ",
+  "Laurence Madrelle",
+  "Leo Lionni",
+  "Leonardo Sonnoli",
+  "Le\u00f3n Mu\u00f1oz Santini",
+  "Leslie Smolan",
+  "Les Mason",
+  "Lester Beall",
+  "Lex Reitsma",
+  "Xiaoxiang Liu",
+  "Liz\u00e1 Defossez Ramalho",
+  "Lou Dorfsman",
+  "Louis Danziger",
+  "Louise Fili",
+  "Louis Gagnon",
+  "Louis Silverstein",
+  "Lucienne Roberts",
+  "Lucille Tenazas",
+  "Ludovic Balland",
+  "Luke Hayman",
+  "Lu Yu",
+  "Maciej Urbaniec",
+  "Majid Abbasi",
+  "Makoto Nakamura",
+  "Malcolm Grear",
+  "Manuel Warosz",
+  "Mara Soler Guitian",
+  "Marc Catal\u00e0",
+  "Marcel Jacno",
+  "Margaret Calvert",
+  "Marian Bantjes",
+  "Mariano Sigal",
+  "Marina Willer",
+  "Mario Eskenazi",
+  "M\u00e1rio Feliciano",
+  "Marion Deuchars",
+  "Mark Gowing",
+  "Markus Dressen",
+  "Markus Weisbeck",
+  "Mark Zeugin",
+  "Marte R\u00f6ling",
+  "Martin Gaberth\u00fcel",
+  "Martin Gavler",
+  "Martin Venezky",
+  "Martin Woodtli",
+  "Martti Mykk\u00e4nen",
+  "Masaaki Hiromura"
```

Comment:
missing comma! 😄

Reply:
!! Sorry I didn't check it carefully and even not aware of the result from CI testings 😭